elasticsearch-model/lib/elasticsearch/model/adapters/active_record.rb

```diff
@@ -7,7 +7,7 @@ module Adapter
       module ActiveRecord
 
         Adapter.register self,
-                         lambda { |klass| !!defined?(::ActiveRecord::Base) && klass.ancestors.include?(::ActiveRecord::Base) }
+                         lambda { |klass| !!defined?(::ActiveRecord::Base) && klass.respond_to?(:ancestors) && klass.ancestors.include?(::ActiveRecord::Base) }
```

Comment:
`Multimodel.new` is an instance of Multimodel, and not an instance of Class, hence it doesn't respond to `ancestors`. An alternative approach will be that multimodel would implement:

```ruby
def ancestors
   []
end
```